### DISCUSSION TITLE
Feature Request: Display Current Time

### DISCUSSION THREAD

**Discussion by StevKast** [linuxserver/Heimdall#683] [Ideas]
I would love to be able to display the current time similar to how the Momentum Chrome extension works. This would be a great alternative to the current ability of the search bar.

Here is what Momentum looks like:
![image](https://user-images.githubusercontent.com/17804308/115769373-7ab53b80-a379-11eb-8ecc-3af0dd19c169.png)

Thanks!

**Comment by KodeStar** [comment]
Take a look at this https://github.com/linuxserver/Heimdall/discussions/1281
